### PR TITLE
Dp 26650 show important contact mobile

### PR DESCRIPTION
**Pull request by clairesunstudio**
Mayflower PR: https://github.com/massgov/mayflower/pull/1847/

DP-23158

### CODE REVIEW

**Comment by joeg8612** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against aef1dc3ff040f0f7f92afb14b4ffc54736bc645e
</p>

**Comment by clairesunstudio** [comment]
Contact data is missing --> https://pr2277-ldza8sb4bnrzvx9sawozuxsnnyxmypgp.tugboatqa.com/orgs/massachusetts-registry-of-motor-vehicles

Found out that in openmass the contact data is contructed in the theme preprocessor
```
$variables['org_contact'] = $node->get('field_ref_contact_info_1')->view('contact_us');
```
Does this mean that the contact row contact items are actually using the contact-us template??

**Comment by joeg8612** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Add a changelog YAML file to this PR</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 31485f29ec3045b1a516e1d367238c38cffb0e4e
</p>

**Comment by clairesunstudio** [comment]
Screen Shot 2024-05-31 at 4 04 22 PM

The org contact is rendered by docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--organization-contact-logo.html.twig
It's using an outdated structure - see the data structure the updated template is expecting:
https://github.com/massgov/mayflower/blob/38c3e936085f6884045f8ab4c036d05c8c024fe6/packages/patternlab/styleguide/source/_patterns/03-organisms/contact/contact-row.json
https://github.com/massgov/mayflower/blob/38c3e936085f6884045f8ab4c036d05c8c024fe6/packages/patternlab/styleguide/source/_patterns/03-organisms/contact/contact-row.twig

File: docroot/themes/custom/mass_theme/mass_theme.theme [hunk 1]
```diff
@@ -7235,10 +7235,7 @@ function mass_theme_preprocess_paragraph__social_media(&$variables) {
 function mass_theme_preprocess_paragraph__organization_contact_logo(&$variables) {
   $paragraph = $variables['paragraph'];
   $node = Helper::getParentNode($paragraph);
-
-  $variables['org_contact'] = $node->get('field_ref_contact_info_1')->view('contact_us');
-  $variables['org_logo'] = $node->get('field_sub_brand')->view('logo_link');
```
Comment (clairesunstudio) [inline docroot/themes/custom/mass_theme/mass_theme.theme:7254]: @tom-fleming, we still need to get the the org logo - can you please get the logo? I can pass it in, or here's the new data structure  https://github.com/massgov/mayflower/blob/38c3e936085f6884045f8ab4c036d05c8c024fe6/packages/patternlab/styleguide/source/_patterns/03-organisms/contact/contact-row.json#L10-L15

After:
https://pr2277-ldza8sb4bnrzvx9sawozuxsnnyxmypgp.tugboatqa.com/orgs/governor-maura-healey-and-lt-governor-kim-driscoll
Before:
https://www.mass.gov/orgs/governor-maura-healey-and-lt-governor-kim-driscoll

**Comment by tom-fleming** [comment]
LGTM

**Pull request merged**